tests/installation/logs_from_installation_system.pm

```diff
@@ -36,6 +36,16 @@ sub run {
     my $dasd_path = get_var('DASD_PATH', '0.0.0150');
     select_console 'install-shell';
 
+    # permit ssh root login as it is disabled in "Common Criteria" "System Role" system
+    if (check_var('SYSTEM_ROLE', 'Common_Criteria') && is_sle) {
+        record_soft_failure('workaround for poo#98715');
+        assert_script_run('mount /dev/vda2 /mnt');
+        assert_script_run("sed -i -e 's/PermitRootLogin/#PermitRootLogin/g' /mnt/etc/ssh/sshd_config");
+        assert_script_run('echo PermitRootLogin yes >> /mnt/etc/ssh/sshd_config');
+        assert_script_run('cat /mnt/etc/ssh/sshd_config');
```

Comment:
This line is used for debug? It's not necessary, right?

Reply:
yes